website/src/app/[lang]/[region]/survey/[recipient]/[survey]/questions.ts

```
@@ -375,73 +340,17 @@ export const longEnoughPage = (t: TranslateFunction) => {
-		'southernProvince',
-		'northWestProvince',
-	].map((key) => {
-		return {
-			value: key,
-			text: t('survey.questions.livingLocationChoices.' + key),
-		};
-	});
-
-export const spendingChoices = (t: TranslateFunction) =>
-	['education', 'food', 'housing', 'healthCare', 'mobility', 'saving', 'investment'].map((key) => {
+export const translateChoices = (t: TranslateFunction, choices: any[], choicesTranslationKey: String) =>
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)